core/base-service/base.spec.js

```diff
@@ -316,7 +316,7 @@ describe('BaseService', function() {
   })
 
   describe('ScoutCamp integration', function() {
-    const expectedRouteRegex = /^\/foo\/([^/]+?)\.(svg|png|gif|jpg|json)$/
+    const expectedRouteRegex = /^\/foo\/([^/]+?)\.(svg|json)$/
```

Comment:
While we're fiddling with this, is this also a sensible point in time to introduce extensionless SVG https://github.com/badges/shields/issues/2674 or not?

Reply:
Yea, it's a good point. I was thinking about that, but I'm eager to deploy this so we can find out how the raster server is performing. Extensionless SVG will need a few other changes made including some CDN config tweaks. I'm curious to find out how stable the raster server is and don't really want to hold this up.

Reply:
Ok. Lets stick to changing one thing at a time :+1: